content/2020/11-17-client-credentials-grant.md

```diff
@@ -0,0 +1,40 @@
+---
+applied_at: "2020-11-17"
+applies_to: 
+- api
+is_impactful: false
+is_new_feature: false
+collapse: false
+show_excerpt: true
+release_source_url: ''
+---
+
+# Client Credentials Grant
```

Comment:
```suggestion
# Announcing Client Credentials Grant authentication
```

As-is the title doesn't really explain what is new. Not sure if this is better, open to suggestions.